.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Should we also mention on which specific branch they should test before reporting the issues